web/Services/LandlordService.cs

```diff
@@ -96,4 +99,26 @@ public async Task ApproveLandlord(int landlordId,  BricksAndHeartsUser user)
         landlord.ApprovalAdminId = user.Id;
         await _dbContext.SaveChangesAsync();
     }
+    
+    public async Task<ILandlordService.LandlordRegistrationResult> EditLandlordDetails(LandlordProfileModel editModel)
+    {
+        var landlordToEdit = await _dbContext.Landlords.SingleAsync(l => l.Id == editModel.LandlordId);
+        landlordToEdit.Title = editModel.Title;
+        landlordToEdit.FirstName = editModel.FirstName;
+        landlordToEdit.LastName = editModel.LastName;
+        landlordToEdit.CompanyName = editModel.CompanyName;
+        landlordToEdit.Email = editModel.Email;
+        landlordToEdit.Phone = editModel.Phone;
+
+        _dbContext.Update(landlordToEdit);

```

Comment:
I wouldn't expect this to be necessary, does it not work without it?

My understanding is that it enables Entity Framework's change tracking on an object, but that should be enabled by default for things read from the database unless you do `.AsNoTracking()`

Reply:
Yep! You're right, I guess I just didn't try testing this without this line ever :)